dev/com.ibm.websphere.appserver.features/test/src/com/ibm/ws/feature/tests/VisibilityTest.java

```diff
@@ -275,17 +275,24 @@ public void testVisibility2() {
             fileName = fileName.replace('\\', '/');
             String expectedPathName = "/visibility/" + visibility + "/";
             if ("public".equals(visibility)) {
-                expectedPathName += (featureInfo.getShortName() + '/');
-            }
-            expectedPathName += (featureName + ".feature");
-            if (!fileName.endsWith(expectedPathName)) {
-                errorMessage.append("Found issues with " + featureName + '\n');
-                errorMessage.append("     Feature is not in the expected directory " + expectedPathName + ".\n");
-                errorMessage.append("     The feature's visibility " + visibility + " may not match the directory.\n");
-                if ("public".equals(visibility)) {
+                expectedPathName += (featureInfo.getShortName() + '/') + (featureName + ".feature");
```

Comment:
Slightly confused about what updates to Feature Visibility Tests have to do with admincenter?

Reply:
This was to support the non public features also being in their own directory.